src/containers/Application/Part3.js

```diff
@@ -1,6 +1,169 @@
-import React from 'react'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Dropdown, Select, TextInput } from '../../components/Input'
+import { H1, QuestionHeading } from '../../components/Typography'
+
+const StyledDropdown = styled(Dropdown)`
+  .react-select__control {
+    margin: 0;
+  }
+`
+
+const QuestionContainer = styled.div`
+  margin-bottom: 6em;
+`
+
+const StyledTextInput = styled(TextInput)`
+  margin: 0.5em 0 0;
+`
 
 // form part 3
-export default ({ children }) => {
-  return <div>{children}</div>
+export default () => {
+  // eslint-disable-next-line no-unused-vars
+  const [selectedOption, setSelectedOption] = useState('')
+  const [otherOption, setOtherOption] = useState('')
+  const [showOther, setShowOther] = useState('')
+  const [selectStates, setSelectStates] = useState({
+    multiselect: {
+      option1: false, // LHD / Hack Camp
+      option2: false, // nwHacks
+      option3: false, // cmd-f
+      option4: false, // cmd-f Phases
+      option5: false, // nwPlus Workshop Series
+      option6: false, // nwPlus Boothing
+    },
+  })
```

Comment:
Is it possible to use one object to keep track of the form state?
```suggestion
  const [selectStates, setSelectStates] = useState({
    engagementSource: '',
    eventsAttended: {
      option1: false, // LHD / Hack Camp
      option2: false, // nwHacks
      option3: false, // cmd-f
      option4: false, // cmd-f Phases
      option5: false, // nwPlus Workshop Series
      option6: false, // nwPlus Boothing
    },
  })
```

Reply:
Yes! Have done that here: dffb17f